datasets/1.0/bigcode-the-stack/metadata.json

```diff
@@ -47,25 +47,19 @@
   "name": "bigcode-the-stack",
   "conformsTo": "http://mlcommons.org/croissant/1.0",
   "description": "The Stack contains over 6TB of permissively-licensed source code files covering 358 programming languages. The dataset was created as part of the BigCode Project, an open scientific collaboration working on the responsible development of Large Language Models for Code (Code LLMs). The Stack serves as a pre-training dataset for Code LLMs, i.e., code-generating AI systems which enable the synthesis of programs from natural language descriptions as well as other from code snippets.",
-  "citeAs": "@article{Kocetkov2022TheStack, title={The Stack: 3 TB of permissively licensed source code}, author={Kocetkov, Denis and Li, Raymond and Ben Allal, Loubna and Li, Jia and Mou,Chenghao and Mu\u00f1oz Ferrandis, Carlos and Jernite, Yacine and Mitchell, Margaret and Hughes, Sean and Wolf, Thomas and Bahdanau, Dzmitry and von Werra, Leandro and de Vries, Harm}, journal={Preprint}, year={2022} }",
   "creator": [
     {
-      "@type": "Person",
-      "name": "Harm de Vries",
-      "email": "harm.devries@servicenow.com"
+      "@type": "Organization",
+      "name": "Harm de Vries"
     },
     {
-      "@type": "Person",
-      "name": "Leandro von Werra",
-      "email": "leandro@huggingface.co"
+      "@type": "Organization",
+      "name": "Leandro von Werra"
     }
   ],
-  "keywords": [
-    "crowdsourced",
-    "expert-generated"
-  ],
+  "citeAs": "@article{Kocetkov2022TheStack, title={The Stack: 3 TB of permissively licensed source code}, author={Kocetkov, Denis and Li, Raymond and Ben Allal, Loubna and Li, Jia and Mou,Chenghao and Mu\u00f1oz Ferrandis, Carlos and Jernite, Yacine and Mitchell, Margaret and Hughes, Sean and Wolf, Thomas and Bahdanau, Dzmitry and von Werra, Leandro and de Vries, Harm}, journal={Preprint}, year={2022} }",
```

Comment:
- Keep keywords
- Alphabetical order with `citeAs` before `creator`

Reply:
Manually diff'd to revert